src/runtime.js

```diff
@@ -9,7 +9,7 @@ export default async function(config) {
   let db = await createConnection(config.mongo.connectionString);
   let jobs = kue.createQueue({
     prefix: config.kue.prefix,
-    redis: config.redis
+    redis: config.redis.connectionString
```

Comment:
Yea from what I saw in the kue documentation "redis" can either be a connection string or an object with host, auth, port, etc.